src/site/topic16.rst

```diff
@@ -5,6 +5,82 @@ Topic #16 --- Exceptions
 Runtime Errors
 ==============
 
+Exceptions
+==========
+
+* Exceptions are *exceptional* situations that programs may want to handle
+* There are two types of exceptions
+    * Unchecked
+    * Checked
+
+Unchecked
+---------
+
+* Unchecked exceptions are ``RuntimeException``s
+* These are exceptions that can be thrown during the normal operation of the JVM
```

Comment:
More specifically, they are not type checked. You can think of checked exceptions as a function with multiple return values. The type checked forces you to write code that complies with that return type.